packages/smart-contract/src/network/native-api.ts

```diff
@@ -42,11 +44,11 @@ export async function sendTransaction(json: any, url: string): Promise<TSendTran
     const { data, errors } = response.data;
 
     if (errors) {
-      throw Error(errors)
+      throw errors;
     }
 
-    return data;
+    return data.sendZkapp.zkapp;
   } catch (error) {
-    throw Error(error as any)
+    throw Error(error as any);
```

Comment:
Is this try catch necessary? you literally just catch and throw it.

Reply:
I think it redundant, removed